core/themes/basis/css/component/cards.css

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file
+ * Card listing view on the default home page.
+ */
+
+.l-top .block-views-promoted-cards-block {
+  margin-top: -2rem;/* Negative top margin collapses spacing under header */
+}
+.block-views-promoted-cards-block {
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+}
+
+.block-views-promoted-cards-block .views-view-grid-cols-3 {
+  grid-template-columns: repeat(1, 1fr);
+}
+.view-promoted-cards .views-grid-box {
+  border: none;
+}
+
+.view-promoted-cards .views-grid-box {
+  border-radius: 0.25rem;
```

Comment:
I see this radius on the bottom of the cards, but not the top.  Is that intentional?